### PR TITLE
Add change font button to board formspec

### LINKED ISSUE #4: Add an option to change the (black|green)board font
The current font is unreadable for the most people. 
yl issue: https://gitea.your-land.de/your-land/bugtracker/issues/5250

IMO, a third button would be the easiest and cleanest solution.
more or less related: https://github.com/pyrollo/display_modpack/issues/51

### PR DESCRIPTION
**Pull request by Niklp09**
Closes https://github.com/mt-mods/display_modpack/issues/4

### CODE REVIEW

**Comment by OgelGames** [comment]
IMO the wipe button should be replaced with the font button.

Also, I noticed the text and buttons are a bit too far down. Would be good to fix that.

image

File: boards/init.lua [hunk 1]
```diff
@@ -35,8 +35,8 @@ local function set_formspec(pos)
 	meta:set_string("formspec",
 		"size[6,4]"..default.gui_bg..default.gui_bg_img..default.gui_slots..
 		"textarea[0.5,0.7;5.5,3;display_text;"..FS("Text")..";" .. display_text .. "]"..
-		"button_exit[3,3.5;2,1;ok;"..FS("Write").."]"..
-		"button_exit[1,3.5;2,1;wipe;"..FS("Wipe").."]")
+		"button_exit[1,3.35;2,1;ok;"..FS("Write").."]"..
+		"button[3,3.35;2,1;font;"..FS("Font").."]")
```
Comment (OgelGames) [inline boards/init.lua]:
```suggestion
		"button_exit[1,2.75;2,1;ok;"..FS("Write").."]"..
		"button[3,2.75;2,1;font;"..FS("Font").."]")
```

File: boards/init.lua [hunk 2]
```diff
@@ -35,8 +35,8 @@ local function set_formspec(pos)
 	meta:set_string("formspec",
 		"size[6,4]"..default.gui_bg..default.gui_bg_img..default.gui_slots..
 		"textarea[0.5,0.7;5.5,3;display_text;"..FS("Text")..";" .. display_text .. "]"..
```
Comment (OgelGames) [inline boards/init.lua]:
```suggestion
		"size[6,3.5]"..default.gui_bg..default.gui_bg_img..default.gui_slots..
		"textarea[0.55,0.25;5.5,3;display_text;"..FS("Text")..";" .. display_text .. "]"..
```

**Review by OgelGames — CHANGES_REQUESTED** [review]

**Comment by OgelGames** [comment]
Suggested changes to fix the position of the text area.

The formspec looks like this with the changes: 
image

**Comment by Niklp09** [comment]
The formspec look currently like this. Better IMO
image

**Pull request merged**